docs/advanced/solo-proof-of-stake-voting.md

```diff
@@ -0,0 +1,198 @@
+
+```
+enableticketbuyer=1
+ticketbuyer.votingaddress=DsHotWalletAddressFromVPS
+ticketbuyer.balancetomaintainabsolute=0
+```
+
+---
+
+## Voting
+Setting vote choices for on-chain votes will happen on the VPSs you will set up and we will address that process in a later section. For Politeia votes you will need to cast those from your cool wallet. I will outline the process for doing so using the `politeiavoter` CLI tool.
+
+1. Download the Politeia archive, manifest, and signature files.
+
+`wget https://github.com/decred/decred-binaries/releases/download/v1.3.1/{politeiavoter-linux-amd64-v1.3.1.tar.gz,manifest-politeiavoter-v1.3.1.txt,manifest-politeiavoter-v1.3.1.txt.asc}`
+
+2. Verify the manifest.
+
+`gpg --verify manifest-politeiavoter-v1.3.1.txt.asc`
+
+4. Verify the SHA-256 hash in the manifest matches that of the archive.
+
+`sha256sum politeiavoter-linux-amd64-v1.3.1.tar.gz`
+
+5. Extract the archive.
+
+`tar -xf politeiavoter-linux-amd64-v1.3.1.tar.gz`
+
+6. Enter the `politeiavoter-linux-amd64-v1.3.1` directory.
+
+
+Now to view the various agendas and vote on them you will need to run the following commands. Also remember that for this to work `dcrd` and `dcrwallet` must also be running.
+
+To get a list of active proposals:
+
+`./politeiavoter inventory`
+
+To vote on a proposal:
+`./politeiavoter vote <prop-hash> <yes|no>`
+
+---
+
+# Setting Up "Hot" Voting Wallets
+## VPS Providers
+The first thing you will need is to choose at least one VPS provider. You will find a short list of decent providers below, you may mix and match any number of providers, ideally your VPS should have at least 2 GB of RAM.
+
+* [Scaleway](https://www.scaleway.com/)
+* [Google Cloud Platform (GCP)](https://cloud.google.com/)
+* [Amazon Web Services (AWS)](https://aws.amazon.com/)
+* [OVH](https://www.ovh.com/)
+
+Note: Another consideration is storage space. As you will need to store the complete Decred blockchain which is constantly growing, it is important to keep in mind that while it is currently quite small (~3 GB), it will eventually grow to the point where you may need to upgrade the storage space on your VPS instances.
+
+---
+
+## Configuration
+Once you have set up a VPS you will need to get Decred up and running on it, so that you may generate a voting address. This will be the address which you specify when you purchase tickets, so as to grant the hot wallets voting rights. Follow the steps below to set up your hot wallet. You can then repeat the steps for each additional instance you want to run, or just clone the VPS from the control panel of your provider and select a different geographic location where you would like to host it (each hot wallet needs to use the same seed). It is highly recommended you run at least three hot wallets, one on the East coast of the US, one in Europe, and one in Asia should be adequate, adding an additional one on the West coast of the US might be desireable as welif you're into overkill.
```

Comment:
```suggestion
Once you have set up a VPS you will need to get Decred up and running on it, so that you may generate a voting address. This will be the address which you specify when you purchase tickets, so as to grant the hot wallets voting rights. Follow the steps below to set up your hot wallet. You can then repeat the steps for each additional instance you want to run, or just clone the VPS from the control panel of your provider and select a different geographic location where you would like to host it (each hot wallet needs to use the same seed). It is highly recommended you run at least three hot wallets, one on the East coast of the US, one in Europe, and one in Asia should be adequate, adding an additional one on the West coast of the US might be desireable as well if you're into overkill.
```